Src/Plugins/Tests.Security/TestData/SecurePlaintextSecrets/Inputs/SEC101_007.GitHubAppCredentials.ps1

```diff
@@ -14,15 +32,46 @@
 	deadbeef00deadbeef00deadbeef00testtest
 </GitHubAppSecret>
 
+********************************************************************************
+********************************************************************************
+********************************************************************************
+********************************************************************************
+********************************************************************************
+********************************************************************************
+
 <GitHubId>Iv1.01234567deaddead</GitHubId>
 <GitHubAppSecret>deadbeef00deadbeef00deadbeef00deadbeef</GitHubAppSecret>
```

Comment:
>deadbeef00deadbeef00deadbeef00deadbeef [](start = 17, length = 38)

please make every credential in this test file unique, or you will break when my latest change comes in. :)

Reply:
Done.

---
In reply to: [573360947](https://github.com/microsoft/sarif-pattern-matcher/pull/183#discussion_r573360947) [](ancestors = 573360947)